docker-compose.yml

```diff
@@ -69,7 +69,7 @@ services:
       file: hyrax-webapp/docker-compose.yml
       service: web
     # command left commented for situations where the Gemfile.lock needs to be updated before the app starts
-    # command: bash -l -c "sleep infinity"
```

Comment:
btw was this supposed to be uncommented?

Reply:
nope, should be commented out I think